CoreQualityControlView/src/au/gov/asd/tac/constellation/views/qualitycontrol/QualityControlViewTopComponent.java

```diff
@@ -89,6 +90,17 @@ public void handleComponentClosed() {
         QualityControlAutoVetter.getInstance().removeListener(this);
     }
 
+
+    @Override 
+    public void handleNewGraph(final Graph graph) {
+        qualityControlViewPane.setFirstClick(true); 
+    }
+    
+    @Override 
+    protected void handleGraphChange(final GraphChangeEvent event) {
+        qualityControlViewPane.setFirstClick(true);
+    }
+    
```

Comment:
I don't believe these changes are required.
The focus listener in the View Pane ensures the firstClick variable will always be set to true whenever the View Pane gets focus.